contracts/core/EntryPoint.sol

```diff
@@ -611,13 +618,20 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuard,
             opIndex,
             userOp,
             outOpInfo,
-            requiredPreFund
+            requiredPreFund,
+            verificationGasLimit
         );
 
         if (!_validateAndUpdateNonce(mUserOp.sender, mUserOp.nonce)) {
             revert FailedOp(opIndex, "AA25 invalid account nonce");
```

Comment:
Just a minor issue, why not incorporate the Nonce verification into _validateAccountPrepayment? It seems that doing so would make the responsibilities of each method clearer.

Reply:
I think the method names declare their responsibilities: 
- _validateAccountPrepayment - performs calls validateUserOp, and validates its response (the payment). 
- _validateAndUpdateNonce - performs nonce validation
(yes, we could wrap them both in "account validation), which does nothing but call them both)